hasura.planx.uk/metadata/tables.yaml

```diff
@@ -2130,35 +2128,8 @@
           - notify_personalisation
           - settings
           - slug
-          - submission_email
         filter: {}
         check: null
-- table:
-    name: teams_summary
-    schema: public
```

Comment:
I might well be missing context here, but there's no reference to dropping this table in the PR description. Is this correct?

This table is used for the admin summary page which still requires this table.

Reply:
It should only be the `submission_email` col, I'll revert this change.

Reply:
@jessicamcinchak @DafyddLlyr this got removed due to the script for creating it having a reference to `teams.submission_email` - I am making a new PR as it's easier for reverting the changes. Sorry for missing it and thanks for picking it up.

https://github.com/theopensystemslab/planx-new/pull/3640